ExampleApps/SwiftUIExample/Views/LoginView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  LoginView.swift
+//  LoginView
+//
+//  Created by Richard Gist on 8/25/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//  swiftlint:disable closure_body_length
+
+import SwiftUI
+import SwiftCurrent
+import SwiftCurrent_SwiftUI
+
+struct LoginView: View, FlowRepresentable {
+    let inspection = Inspection<Self>() // ViewInspector
+    weak var _workflowPointer: AnyFlowRepresentable?
+    @State var email = ""
+    @State var password = ""
+    @State var showSignUp = false
+    @State var showPassword = false
+    @State var isLoggedIn = false
+
+    var body: some View {
+        GeometryReader { _ in
+            VStack(spacing: 40) {
+                Image.logo
+                    .resizable()
+                    .shadow(color: .white.opacity(0.1), radius: 5, x: 0, y: 5)
+                    .frame(width: 120, height: 120)
+                    .foregroundColor(.icon)
+
+                ZStack(alignment: .bottom) {
+                    VStack {
+                        Text("Welcome!")
+                            .titleStyle()
+
+                        VStack(spacing: 15) {
+                            PrimaryTextField(placeholder: "Email Address", image: Image.account, text: $email)
+                                .keyboardType(.emailAddress)
+
+                            PasswordField(showPassword: $showPassword, password: $password)
+                        }
+
+                        HStack {
```

Comment:
? An HStack with only a spacer in it? I'm sure there's a reason for this but I can't fathom what it is.